jpademo/src/main/java/com/likelion/jpademo/entity/Post.java

```diff
@@ -0,0 +1,41 @@
+package com.likelion.jpademo.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@NoArgsConstructor
```

Comment:
여러 가지 이유로 JPA 엔티티에도 접근제어자를 설정해줍니다! 이유는 가은님께서 찾아보시면 좋을 것 같아요~

```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```